docs/UserGuide.md

```diff
@@ -86,57 +83,64 @@ Format for adding a new asset: `add --id <asset id> --location <location>`
 * Any number of digits are allowed.
 * ‘+’ is optional and must be the first character.
 
-`--tag`
-* Tag associated with contact.
-* Contact can be created first without tags, then tags can be added later.
-* Case sensitive, i.e. NUS != nus.
-* Must have no spaces.
-
-`--id`
-* An ID associated with the asset.
-* Not necessarily unique.
+`--address`
+* Address of the contact.
 
-`--location`
-* The location associated with the asset.
+`--asset`
+* Asset(s) associated with contact.
+* Contact can be created first without assets, then assets can be added later using the Edit command.
+* Case sensitive, i.e. NUS ≠ nus.
+* Assets must have unique names. If the asset already exists in the database, the existing asset will be linked instead of a new asset.
+* Multiple assets can be specified. For example, a valid option is `--asset asset1 --asset asset2 --asset asset3`.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ### Delete a contact or asset : `delete`
 
-Delete a contact or asset from the system by specifying its ID.
+Delete a contact from the system by specifying its ID.
 
 Format: `delete <id>`
-* The id refers to the unique id shown in the GUI.
+* The id refers to the unique contact id shown in the GUI.
+* The asset(s) associated with the contact will not be deleted.
```

Comment:
Should this be the case?

Reply:
Yes, the asset would remain in the `HashMap`, so we don't need to track the number of people using each asset.